test/e2e/login-helpers.js

```diff
@@ -39,6 +39,10 @@ function initUserMock(token, level) {
             gender: 'F',
             birth_date: '1985-01-01'
           },
+          veteran_status: {
+            status: 'OK',
```

Comment:
Is there a list somewhere of all the possible statuses?

Reply:
These look like the common vets-api error statuses defined [here](https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/common/client/concerns/service_status.rb) and shared by [EVSS responses](https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/evss/response.rb#L26) and [MVI responses](https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/mvi/responses/find_profile_response.rb#L60):  OK, NOT_FOUND, SERVER_ERROR, NOT_AUTHORIZED

These should probably be defined in a constants.js file, perhaps in src/js/common/utils

Sitewide ticket?

Reply:
Sounds like a site-wide ticket to me. 👍

Reply:
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4849